### PR TITLE
Fix grammar and capitalization

### PR DESCRIPTION
**Pull request by mbrukman**
Also add missing commas, code formatting, etc.

### CODE REVIEW

File: content/en/guide/v10/getting-started.md [hunk 1]
```diff
@@ -125,16 +125,16 @@ To transpile JSX you need a babel plugin that converts it to valid JavaScript co
...
-> [babeljs](https://babeljs.io/) has some of the best documentation out there. We highly recommend checking it out for questions surrounding babel and how to set it up.
+> [Babel](https://babeljs.io/) has some of the best documentation out there. We highly recommend checking it out for questions surrounding Babel and how to set it up.
 
 ### Aliasing React to Preact
 
-At some point you'll probably want to make use of the vast react ecosystem. Libraries and Components originally written for React work seamlessly with our compatibility layer. To make use of it we need to point all `react` and `react-dom` imports to Preact. This step is called aliasing.
+At some point, you'll probably want to make use of the vast React ecosystem. Libraries and Components originally written for React work seamlessly with our compatibility layer. To make use of it, we need to point all `react` and `react-dom` imports to Preact. This step is called _aliasing._
 
-#### Aliasing in webpack
+#### Aliasing in Webpack
```
Comment (developit) [inline content/en/guide/v10/getting-started.md:134]: [they're picky about this one](https://webpack.js.org/branding/#the-name) :P

```suggestion
#### Aliasing in webpack
```

File: content/en/guide/v10/getting-started.md [hunk 2]
```diff
@@ -125,16 +125,16 @@ To transpile JSX you need a babel plugin that converts it to valid JavaScript co
...
 
-To alias any package in webpack you need to add the `resolve.alias` section
-to your config. Depending on the configuration you're using this section may
+To alias any package in Webpack, you need to add the `resolve.alias` section
```
Comment (developit) [inline content/en/guide/v10/getting-started.md:136]:
```suggestion
To alias any package in webpack, you need to add the `resolve.alias` section
```

File: content/en/guide/v10/getting-started.md [hunk 3]
```diff
@@ -186,11 +186,11 @@ module.exports = {
...
 
-#### Aliasing in jest
+#### Aliasing in Jest
 
-Similar to bundlers, [jest](https://jestjs.io/) allows to rewrite module paths. The syntax is a bit
-different, than in say webpack, because it's based on regex. Add this to your
-jest configuration:
+Similar to bundlers, [Jest](https://jestjs.io/) allows to rewrite module paths. The syntax is a bit
+different than in Webpack, for example, because it's based on regex. Add this to your
+Jest configuration:
```
Comment (developit) [inline content/en/guide/v10/getting-started.md:193]:
```suggestion
[Jest](https://jestjs.io/) allows the rewriting of module paths similar to bundlers.
These rewrites are configured using regular expressions in your Jest configuration:
```

**Comment by mbrukman** [comment]
@developit — thanks for the review! Applied suggested fixes; please take a look.

**Pull request merged**